client/src/app/site/motions/services/motion-multiselect.service.ts

```diff
@@ -24,6 +31,14 @@ import { ViewMotion } from '../models/view-motion';
     providedIn: 'root'
 })
 export class MotionMultiselectService {
+    private get activeMeetingId(): ActiveMeetingIdService {
```

Comment:
Make and resutn type does not fit. If the serviceCollector has the `activeMeetingIdService`, than there is a missing getter in one of the base classes (the base class, which required the service collector). Please add it there. Same for the action service.